packages/platform-core/src/object_accumulator.ts

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
```

Comment:
Since this is now a dependency of client-config which is used in the browser, we may want to only pull in the specific lodash functions that we use. They publish each function as a separate package on npm, eg https://www.npmjs.com/package/lodash.mergewith

Reply:
done